src/eventlistener.cpp

```diff
@@ -96,24 +122,32 @@ bool EventListener::checkOtherInstances(const QString& windowName) {
   }
 #endif
 
+  // Try to wake the UI and bring it to the foreground.
   logger.debug() << "Try to communicate with the existing instance";
```

Comment:
Should the old log be retained?
  logger.debug() << "Terminating the current process";